tests/integration/vxlan_test.py

```diff
@@ -22,16 +22,19 @@
 import pytest
 
 import libnmstate
-
 from libnmstate.error import NmstateVerificationError
 from libnmstate.schema import Interface
 
 from .testlib import assertlib
 from .testlib.bondlib import bond_interface
+from .testlib.cmd import exec_cmd
+from .testlib.cmd import RC_SUCCESS
+from .testlib.cmd import format_exec_cmd_result
```

Comment:
Please sort it. You could also use `from .testlib import cmd`, it will have a nice short context. Up to you.